app/views/tag/_tagging.html.erb

```diff
@@ -1,20 +1,57 @@
-<h1>Tags</h1>
-<em class="italics"><a href="/tags">Tags</a> organize our knowledge base. Click to learn more or subscribe to a topic.</em>
+<!-- This is the sidebar tagging display, also renders Subscribe button for multiple subscription -->
+<div>
+  <h1 style="display: inline-block;">Tags</h1>
+  <% if @node %>
+    <%= render partial: 'tag/subscribe_button', locals:{tags: @node.node_tags} %>
+  <% else %>
+    <%= render partial: 'tag/subscribe_button', locals:{tags: @user.user_tags} %>
+  <% end %>
+</div>
+  <em class="italics"><a href="/tags">Tags</a> organize our knowledge base. Click to learn more or subscribe to a topic.</em>
 
+  <div style="background: #f2f2f2; padding-left: 10px;border-radius: 10px; margin-top: 10px">
```

Comment:
Unless I'm mistaken, everything from this line down should be reverted to how it was originally; as we're dealing with everything for this PR isolated into the above partials. Thank you!

Reply:
the return to thing is exactly what I did, I just didn't push the changes because I wasn't sure. Thanks for confirming that, I am going to push it now!

Reply:
Oh actually this isn't very new, its just the well design for tags in side bar, you want me to remove all this and restore it back to the way it was before?

Reply:
Yes I think Jeff is saying to restore it.